### ISSUE TITLE
Problem caused by about-time update

### ISSUE THREAD

**Issue by tposney** [bitkiller0/spellTemplateManager#23]
Firstly apologies. About time now depends on Simple Calendar and no longer does time/date maniuplations (there is a compatibility layer to make migration easier). I did not realise that anyone was using
```
game.settings.get("about-time", "seconds_per_round")
at registerSpellTemplateManagerSettings (settings.js:149)
```
and so had removed it. The "correct" way to do this going forward is to use the value in:
```
CONFIG.time.roundTime
```
Again, sorry for causing the issue.

**Comment by bitkiller0** [comment]
Due to additional changes in the about time module, this will take additional coating to resolve.

**Comment by bitkiller0** [comment]
This issue is corrected in 2.1.2.
